File: back/Scoreboard/Models/Entities/Player.cs
namespace Scoreboard.Models.Entities;

public class Player
{
    public int Id { get; set; }
    public int? Number { get; set; }
    public string Name { get; set; } = string.Empty;
    public int TeamId { get; set; }

    // 🔗 relación
    public Team? Team { get; set; }
}

File: app/Models/Player.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Player extends Model
{
    use HasFactory;

    protected $fillable = [
        'name',
        'email',
        'age',
        'team',
        'position',
        'number',
        'nationality',
    ];

    protected $casts = [
        'age' => 'integer',
        'number' => 'integer',
    ];
}
